src/components/layout/AppLayout.tsx

```diff
@@ -53,12 +53,21 @@ const styles = (theme: { workspaces: { drawer: { width: any } } }) => ({
     // width: `calc(100% + ${theme.workspaces.drawer.width}px)`,
     width: '100%',
     transition,
+  },
+  appContentTransformTextWorkspace: {
     transform() {
       return workspaceStore.isWorkspaceDrawerOpen
         ? 'translateX(0)'
         : `translateX(-${theme.workspaces.drawer.width}px)`;
     },
   },
+  appContentTransformIconWorkspace: {
+    transform() {
+      return workspaceStore.isWorkspaceDrawerOpen
```

Comment:
same here.

Reply:
I'm not sure that it is more readable. 
If it's ok for you, I can push that change

```javascript
  appContentTransformTextWorkspace: {
    transform() {
      return `translateX(${
        workspaceStore.isWorkspaceDrawerOpen
          ? '0'
          : `-${theme.workspaces.drawer.width}px`
      })`;
    },
  },
  appContentTransformIconWorkspace: {
    transform() {
      return `translateX(${
        workspaceStore.isWorkspaceDrawerOpen ? '0' : `-75px`
      })`;
    },
  },
```